client/a8c-for-agencies/components/a4a-number-input/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, TextControl } from '@wordpress/components';
+import { Icon, lineSolid, plus } from '@wordpress/icons';
+import { useCallback } from 'react';
+
+import './style.scss';
+
+type Props = {
+	value: number;
+	onChange: ( value: number ) => void;
+	minimum?: number;
+	maximum?: number;
+	increment?: number;
+};
+
+export default function A4ANumberInput( {
+	value,
+	onChange,
+	minimum = 1,
+	maximum,
+	increment = 1,
+}: Props ) {
+	const onDecrement = useCallback( () => {
+		onChange( Math.max( value - increment, minimum ) );
+	}, [ increment, minimum, onChange, value ] );
+
+	const onIncrement = useCallback( () => {
+		onChange( maximum ? Math.min( value + increment, maximum ) : value + increment );
+	}, [ increment, maximum, onChange, value ] );
+
+	return (
+		<div className="a4a-number-input">
+			<Button onMouseDown={ onDecrement }>
+				<Icon className="gridicon" icon={ lineSolid } size={ 18 } />
```

Comment:
Can we name this something like `a4a-number-input__icon` instead of gridicon? Also, is this not being used anywhere? Is this done to apply some styles from the gridicon?

Reply:
Good catch! The class name is not needed and was supposed to be removed. 👍